app/lib/lnd/lightning.js

```diff
@@ -94,6 +94,21 @@ class Lightning {
     }
   }
 
+  /**
+   * Gracegfully shutdown the gRPC service.
+   */
+  shutdown() {
+    this.unsubscribe()
+    return new Promise((resolve, reject) => {
+      this.lnd.stopDaemon({}, (err, data) => {
```

Comment:
can we get a `log` here or somewhere so it's easier to see this gets called? Tough to test/debug. Only relevant log is `Shutting down lightning Wallet...`. Would be nice to see a `Calling lnd method: { msg: 'stopDaemon', data: undefined }` or something like that